package.json

```diff
@@ -9,9 +9,9 @@
   },
   "scripts": {
     "start": "node main.js",
-    "test": "mocha -w",
+    "test": "mocha --recursive -w",
```

Comment:
why recursive? because you added folders under test/ ?

Reply:
Chyeas, yeah

Reply:
We can talk about setting up the tests into modules and importing them from the subfolder out into the main `PlayerActions.test.js` but there are some hacks to get that working cleanly / in a way that would make sense for the tests